app/views/admin/admin_portals/index.html.erb

```diff
@@ -1,5 +1,12 @@
   <div class="grid gap-2 m:gap-4 l:gap-6 m:grid-cols-2 l:grid-cols-3 px-2 m:px-0">
-    <% Admin::ApplicationController::MENU_ITEMS.each do |menu_item| %>
+    <%# Temporary code while we use a feature flag for the profile admin. %>
```

Comment:
This is not pretty. BUT: dealing with frozen constants is a bit painful and I didn't feel like adding a whole new helper for something that's hopefully quite temporary. If people prefer that I can extract it though.